share/eds/eds.go

```diff
@@ -28,16 +29,19 @@ import (
 	"github.com/celestiaorg/celestia-node/share/ipld"
 )
 
+// childHashLen is the length of each half of an inner node's data. It consists of the respective
+// min and max namespaces, followed by a sha256 hash.
+const childHashLen = (appconsts.NamespaceSize * 2) + sha256.Size
```

Comment:
I wonder if we should exposes `ipld.nmtHashSize` and use it here, it is the same.

Reply:
+ for exposing is that I use it in `shrex/nd`. So that would be 2 usages already

Reply:
good find, done